howso/engine/trainee.py

```diff
@@ -3584,6 +3572,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def get_contribution_matrix(
         self,
         features: t.Optional[Iterable[str]] = None,
+        directional: bool = False,
```

Comment:
Might be a good time to make all these parameters kwarg only. Adding directional in the middle could break anything currently using positional arguments.

Reply:
I like that idea.